SourceX/DiabloUI/art.cpp

```diff
@@ -3,7 +3,7 @@
 
 namespace dvl {
 
-void LoadArt(const char *pszFile, Art *art, int frames, SDL_Color *pPalette)
+void LoadArt(const char *pszFile, Art *art, int frames, SDL_Color *pPalette, int frame_height)

```

Comment:
plz make a variant that takes frame height instead of frames, it's a bit odd to have it take both.

Reply:
what's wrong with current optional argument? I wanted to make the diff as small as possible ;p

Reply:
Sometimes it's nicer to have it as clean as possible :)

I'm actually thinking that the best option is to split it into an array of single portraits since this will allow adding new ones without editing the others.

Reply:
I'm thinking the best thing is to use stick figures instead for portraits as it makes it more flexible when adding new classes.

Reply:
![image](https://user-images.githubusercontent.com/204594/111157454-2effbd00-8597-11eb-9a94-b7dd071bc6eb.png)